.travis.yml

```diff
@@ -334,7 +334,7 @@ jobs:
         - bash ./scripts/docker-run-memory-test.sh
 
     # Sanitizer tests
-    - if: (type = cron) OR (branch = prerelease_test)
+    - if: (type = pull_request) OR (type = cron) OR (branch = prerelease_test)
```

Comment:
NOTE: sanitizer tests currently take around 22 minutes to run
(personally, I'd kinda like then to run on every PR, but I'm not sure we have the time-budget)

Reply:
Removed in the latest commit.